src/test/features/response/routes/defence-options.ts

```diff
@@ -35,6 +36,7 @@ describe('Defendant response: defence options page', () => {
       })
 
       checkAlreadySubmittedGuard(app, 'get', defenceOptionsPage)
+      checkCountyCourtJudgmentRequestedGuardGuard(app, 'get', defenceOptionsPage)
```

Comment:
We should also be testing that query is applied to POST requests.

Reply:
In that case I'd also update tests for `checkAlreadySubmittedGuard`. Do you agree?